dev/io.openliberty.checkpoint.springboot.crac.app/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.checkpoint.springboot.crac.app</name>
```

Comment:
Using the naming of of other spring boot test applications I suggest we name this `io.openliberty.checkpoint.springboot.fat30.app`